src/jsrs_parser.cc

```diff
@@ -150,6 +151,11 @@ static bool GetType(const char* begin, const char* end, Type* type) {
       }
       break;
     }
+    case 'N':
+    case 'I': {
+      *type = Type::kNumber;
```

Comment:
Oops, I am sorry, I was really talking about `atof` in the previous comment, not `strtol`.
`strtod` has to be used instead of it.